lib/src/booleans/boolean_contains.dart

```diff
@@ -0,0 +1,194 @@
+import 'package:turf/src/invariant.dart';
+import 'package:turf/turf.dart';
+
+import 'boolean_point_in_polygon.dart';
+import 'boolean_point_on_line.dart';
+
+/// [booleanContains] returns [true] if the second geometry is completely contained
+/// by the first geometry.
+/// The interiors of both geometries must intersect and, the interior and
+/// boundary of the secondary must not intersect the exterior of the primary.
+/// Boolean-contains returns the exact opposite result of the `turf/boolean-within`.
+/// example:
+/// ```dart
+/// var line = LineString(coordinates: [
+///   Position.of([1, 1]),
+///   Position.of([1, 2]),
+///   Position.of([1, 3]),
+///   Position.of([1, 4])
+/// ]);
+/// var point = Point(coordinates: Position.of([1, 2]));
+/// booleanContains(line, point);
+/// //=true
+/// ```
+bool booleanContains(GeoJSONObject feature1, GeoJSONObject feature2) {
+  var geom1 = getGeom(feature1);
+  var geom2 = getGeom(feature2);
+
+  var coords1 = (geom1 as GeometryType).coordinates;
+  var coords2 = (geom2 as GeometryType).coordinates;
+  Exception _exception() =>
+      Exception("{feature2 $geom2 geometry not supported}");
+  if (geom1 is Point) {
+    if (geom2 is Point) {
+      return coords1 == coords2;
+    } else {
+      throw _exception();
+    }
```

Comment:
I think this should be  `final _exception = Exception(...);` instead of a function